spec/models/canonical/sync/alchemical_properties_spec.rb

```diff
@@ -30,9 +30,12 @@
       end
 
       context 'when there are no existing records in the database' do
-        it 'populates the models from the JSON file' do
+        it 'populates the models from the JSON file', :aggregate_failures do
           perform
-          expect(AlchemicalProperty.count).to eq 4
+          expect(AlchemicalProperty.find_by(name: 'Cure Disease')).to be_present
```

Comment:
An attentive reader will notice that, in some of the spec files, it's just `expect { perform }.to change(Model, :count).from(0).to(4)` and in some, such as this one, the specific models are actually expected. The reason for this is that, in the case of models that have associations, the existence of the specific models is implicitly tested in the tests for their associations. When models don't have associations, there aren't other specs where the model's existence is tested so we need to test for them explicitly.